word2vec_decade_experiment/03_word_decade_figure_generator_2000_2020.py

```diff
@@ -102,17 +110,17 @@
 local_distance
 
 g = plot_token_timeline(token_timeline_low_dim_df)
-g.save("output/figures/are_time_plot.png")
+g.save("output/timeline_figures/are_time_plot.png")
 print(g)
 
 plot_wordcloud_neighbors_gif(
-    token_timeline_low_dim_df, plot_filename="output/wordcloud_plots/are.gif"
+    token_timeline_low_dim_df, plot_filename="output/wordcloud_plots_2000_2020/are.gif"
```

Comment:
Since you are reusing this folder, it might be better to put that in a variable.